cime_config/allactive/config_compsets.xml

```diff
@@ -353,6 +353,16 @@
   <lname>1950SOI_EAM%CMIP6_ELM%SPBC_MPASSI%DIB_MPASO%IBDISMF_MOSART_SGLC_SWAV</lname>
 </compset>
 
+<compset>
+  <alias>CRYO1850-TMIX</alias>
+  <lname>1850SOI_EAM%CMIP6_ELM%SPBC_MPASSI%DIB_MPASO%IBPISMFTMIX_MOSART_SGLC_SWAV</lname>
+</compset>
+
+<compset>
+  <alias>CRYO1950-TMIX</alias>
+  <lname>1950SOI_EAM%CMIP6_ELM%SPBC_MPASSI%DIB_MPASO%IBPISMFTMIX_MOSART_SGLC_SWAV</lname>
+</compset>
+
```

Comment:
We would presumably remove these for now and use the long names if needed.

Reply:
Deleted these